core/src/main/java/jenkins/util/JSONSignatureValidator.java

```diff
@@ -220,10 +224,10 @@ private boolean verifySignature(Signature signature, String providedSignature) {
         }
 
         try {
-            if (signature.verify(Base64.decode(providedSignature.toCharArray()))) {
+            if (signature.verify(Base64.getDecoder().decode(providedSignature))) {
                 return true;
             }
-        } catch (SignatureException|IOException ignore) {
+        } catch (SignatureException ignore) {
```

Comment:
Should it be also IllegalStateException here?

Reply:
IllegalStateException is also RuntimeException, Do we want to capture it?

Reply:
maybe

Reply:
done